### DISCUSSION TITLE
realtime YOLOv5 in web browser

### DISCUSSION THREAD

**Discussion by nihui** [ultralytics/yolov5#6749] [Show and tell]
# open https://nihui.github.io/ncnn-webassembly-yolov5 and enjoy

hi, I have implemented realtime yolov5 object detection in web browser with ncnn and webassembly

source here  --->  https://github.com/nihui/ncnn-webassembly-yolov5

The model used is yolov5n v6.1, it shall be compatible with all modern web browser across desktop and mobile.

**Comment by glenn-jocher** [comment]
@nihui awesome and very fast (both uploading the new v6.1 model and the FPS!).

**Comment by kalenmike** [comment]
@nihui Awesome! Great work!
